Makefile

```diff
@@ -275,3 +284,11 @@ scratch: $(RPM_SPECFILE) $(RPM_TARBALL)
 		--nocheck \
 		$(RPM_SPECFILE)
 
+container_composer_golangci_built.info: Makefile Containerfile_golangci_lint tools/install_golangci_lint_deps.sh
+	podman build -f Containerfile_golangci_lint -t $(GOLANGCI_COMPOSER_IMAGE) --build-arg "GOLANGCI_LINT_VERSION=$(GOLANGCI_LINT_VERSION)"
+	echo "Image last built on" > $@
+	date >> $@
+
+.PHONY: lint
+lint: $(GOLANGCI_LINT_CACHE_DIR) container_composer_golangci_built.info
+	podman run -t --rm -v $(SRCDIR):/app:z -v $(GOLANGCI_LINT_CACHE_DIR):/root/.cache:z -w /app $(GOLANGCI_COMPOSER_IMAGE) golangci-lint run -v
```

Comment:
I'm not entirely sure what benefit this has over just installing (or downloading) the golangci-lint version we care about and running it without a container.  I'm not against it, but I personally will never run this instead of `go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.53 run`.

Reply:
The environment would be more similar to the GH workflow, which is the one rejecting your PR later on 🤔

Reply:
I guess that's fair.